src/SimplePie.php

```diff
@@ -2757,7 +2757,7 @@ public function get_image_link()
     public function get_image_width()
     {
         if ($return = $this->get_image_tags(self::NAMESPACE_RSS_20, 'width')) {
-            return round(intval($return[0]['data']));
+            return intval($return[0]['data']);
```

Comment:
I was thinking moving the `round` inside `intval` so that the value is int but properly rounded.

Reply:
Yeah, I would assume that is why `round` was used in the first place.

Reply:
But maybe it is not worth it.

Reply:
Before we can use `round()` we would convert it to float first. It would look like this: 
```php
return intval(round(floatval($return[0]['data'])));
```

see also: https://3v4l.org/sHMLb

I don't think (and hope) that anybody would provide a size as "half pixels". 😄 Or at least would don't have to support this. According to [RSS 2.0 spec](https://www.rssboard.org/rss-specification#ltimagegtSubelementOfLtchannelgt) the pixels must be "numbers".